pkg/osv/osv.go

```diff
@@ -319,44 +314,56 @@ func HydrateWithClient(resp *BatchedResponse, client *http.Client) (*HydratedBat
 	return &hydrated, nil
 }
 
+// makeRetryRequest will return an error on both network errors, and if the response is not 200
 func makeRetryRequest(action func() (*http.Response, error)) (*http.Response, error) {
 	var resp *http.Response
 	var err error
 	for i := 0; i < maxRetryAttempts; i++ {
+		time.Sleep(time.Duration(i*i) * time.Second)
```

Comment:
should we add some random jitter to this?

Reply:
nit: might also make a little more logical sense to have the sleep after the actual action. particularly if we add jitter.

Reply:
added some random jitter. I think it makes more sense at the start than the end, as this avoids having to add a if statement to check if this is the last loop, and avoids having to add `i+1` as well.